otel-tests/src/lib.rs

```diff
@@ -22,9 +19,23 @@ pub use otel_tests_macro::otel_test;
 static INIT_OTLP_EXPORTER: Once = Once::new();
 
 pub fn init_tracing(module: &'static str) {
+    let env = std::env::var("ITCHYSATS_TEST_INSTRUMENTATION").unwrap_or_default();
```

Comment:
do we still need the envvar? It seems that if we compile it with support, we might as well use it? (it's already opt-in)

Reply:
That's true. I left it in for now, as you might want to ignore the output for one run or something without recompiling everything. I can remove the env var too, though.

Reply:
might as well keep it for now, until it bugs someone :-)